### ISSUE TITLE
ESPCN problem

### ISSUE THREAD

**Issue by suke27** [drakelevy/ESPCN-TensorFlow#3]
hi，when I run espcn model. I found for some images it output bad result.
do you know how to improve it. adjust train dataset ?
Thank you!

**Comment by drakelevy** [comment]
Try training with more data and ensure the data is varied so the network can learn to generalize. Training longer and experimenting with adjusting the learning rate can also help.
